src/common/httpClient/models.ts

```diff
@@ -28,3 +29,13 @@ export interface HttpResult<TDto> {
 export interface BasicHttpClient {
   request<TDto>(config: HttpRequestConfiguration): Promise<HttpResult<TDto>>;
 }
+
+//https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseType
```

Comment:
do we need that comment here?

Reply:
I thought it could be helpful to have a reference here

Reply:
okay, got ya. 

```suggestion
// https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseType
```